zerver/openapi/zulip.yaml

```diff
@@ -12470,6 +12470,97 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /export/realm:
+    get:
+      operationId: get-realm-exports
+      summary: Get all the exports of the organization
+      tags: ["server_and_organizations"]
+      description: |
+        Fetch all the requested [data exports](/help/export-your-organization) of
+        the organization.
+
+        **Changes**: New in Zulip 2.1.
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      exports:
+                        type: array
+                        description: |
+                          An array of dictionaries where each dictionary contains
+                          data about a single organization export request.
+                        items:
+                          $ref: "#/components/schemas/RealmExport"
```

Comment:
Aren't these keyed with the export ID? If so, we have to do our workaround to display that properly. See get all custom emoji endpoint response documentation as a reference for this.

Reply:
No, the exports are not keyed by export ID. The `exports` parameter is an array of data exports sorted by export ID.

Reply:
Ah, yeah you're correct...

```python
return sorted(exports_dict.values(), key=lambda export_dict: export_dict["id"])
```